netci.groovy

```diff
@@ -53,7 +53,7 @@ class Constants {
                'jitstress2_jitstressregs0x10' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x10'],
                'jitstress2_jitstressregs0x80' : ['COMPlus_JitStress' : '2', 'COMPlus_JitStressRegs' : '0x80'],
                'corefx_baseline' : ['' : ''], // corefx baseline
```

Comment:
Ahh in this case, you would do: [:] for an empty map, avoiding the odd k = '' case.